### DISCUSSION TITLE
[Debian Package] Asking about mdurl dependency

### DISCUSSION THREAD

**Discussion by eamanu** [executablebooks/meta#702] [Q&A] [answered]
Hi!

I'm working on the packaging of markdown-it-py v2.0.1 in Debian. One of the new dependency
in 2.0.0 is mdurl. Looking for it, I can note that in Pypi [0], the homepage redirect to @hukkin 's 
fork [1] from [2]. 

mdurl, is not part of Debian, my plan is package it, but I don't know which fork take. Actually I
usually use the Git upstream repository for packaging, but in this case if I'm not wrong 
markdown-it-py build take the pypi package (so @hukkin 's fork).

Should I use the fork or the executablebooks repo? 

Thank you very much!

PS: thanks for your work, markdown-it-py is amazing.

[0] https://pypi.org/project/mdurl/
[1] https://github.com/hukkin/mdurl
[2] https://github.com/executablebooks/mdurl

**Comment by hukkin** [comment, marked as answer]
Hi! Please use the executablebooks repo. The link on PyPI will be fixed in the next release. The reason why the link currently points to my fork is that the repository was initially mine, then it was transferred to executablebooks, and after that I've forked the executablebooks repo (this last step unfortunately breaks the redirect that GitHub automatically creates when transferring a repo).

**Reply by eamanu** [reply]
Thanks for clarify to me that.
